content/en/security/cloud_security_management/guide/jira.md

```diff
@@ -28,7 +28,7 @@ To create a Jira issue for one or more resources impacted by a misconfiguration:
 
 1. On the [Misconfigurations Explorer][1], select a misconfiguration.
 2. Under **Resources Impacted**, select one or more resources.
-3. On the **Actions** dropdown menu, select **Create Jira Issue**.
+3. On the **Actions** dropdown menu that appears on top, select **Create Jira Issue**.
```

Comment:
I'm adding this detail because I think it's very easy to confuse these two buttons
![Screenshot 2023-11-14 at 16 52 15](https://github.com/DataDog/documentation/assets/122295290/d4b5258d-07ec-4208-a907-a709f4c1a010)